lib/API/JSON/v1/Mailer.php

```diff
@@ -40,7 +45,10 @@ function send($data = []) {
         (isset($data['sender'])) ? $data['sender'] : false,
         (isset($data['reply_to'])) ? $data['reply_to'] : false
       );
-      $result = $mailer->send($data['newsletter'], $data['subscriber']);
+      $extra_params = [
+        'meta' => $this->mailerMetaInfo->getSendingTestMetaInfo(),
```

Comment:
This looks like a generic sending API class. I don't think we can hardcode here "sending test" meta info. Can we?

Reply:
I checked the code and found that we only call it to send the test email on from the settings page. I know that this may change in the future, but then what do you suggest as metadata for this email?

Reply:
@amine-mp Yeah, it is kind of dangerous to suppose this is called for tests only. Maybe we should extend the API with some parameters? Or maybe we can ask what others think on Slack?